packages/util/json/index.ts

```diff
@@ -10,5 +10,20 @@ export const replacer = {
       return number.toHex(v)
     }
     return v
+  },
+  uint8ArrayToHexString: (_: string, v: any) => {
+    // Filtering out properties
+    if (v instanceof Uint8Array) {
+      return Buffer.from(v).toString("hex")
+    }
+    return v
   }
 }
+
+export function format(
+  data: any,
+  replacer: (this: any, key: string, value: any) => any | null = null
+): string {
+  const json = JSONB({ useNativeBigInt: true })
```

Comment:
And also, make the default export as const and reuse here, instead of reconstructing it

```typescript
const json = JSONB({ useNativeBigInt: true })

export default json

export function format(...) {
    return json.stringify(...)
}
```